openlcsd/flow/tasks.py

```diff
@@ -944,27 +944,48 @@ def get_module_components_from_corgi(context, engine):
     engine.logger.info("Start to get module components data...")
     mc = ParentComponentsAsync(
         config.get('CORGI_API_PROD'), context.get('package_nvr'))
-    context['components'] = mc.get_components_data('RHEL_MODULE')
+    context['components'] = mc.get_components_data('RPMMOD')
     engine.logger.info("Finished getting module components data.")
 
 
+def get_module_srpm(context, engine):
+    """
+    The get_module_components_from_corgi gets the RPM components that includes
+    the rpms and srpms. This function gets the srpm builds for rhel module.
+    """
+    components = context.get('components')
+    rpm_components = components.get('RPM')
+    engine.logger.info("Start to get srpm builds of the module...")
+    srpms = []
+    for rpm in rpm_components:
+        if rpm.get('arch') == 'src':
+            nvr = [rpm.get('name'), '-', rpm.get('version'), '-',
+                   rpm.get('release')]
+            srpms.append(''.join(nvr))
```

Comment:
It is better to change the code as follows:
```
            nvr = "-".join([rpm.get('name'), rpm.get('version'), rpm.get('release')])
            srpms.append(nvr)
```

Reply:
Sure, will change.